resources/main.mjs

```diff
@@ -42,6 +42,15 @@ class MainBenchmarkClient {
         }
 
         this._developerModeContainer?.remove();
+        this._progressCompleted = document.getElementById("progress-completed");
+        if (params.iterationCount < 50) {
+            const progressNode = document.getElementById("progress");
+            for (let i = 1; i < params.iterationCount; i++) {
+                const iterationMarker = progressNode.appendChild(document.createElement("div"));
```

Comment:
Wouldn't this be a great use case for https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress ? :-)

Reply:
Ah didn't know about that one. I'll try to add that next week then :).